config/s2i/gold/continuous-infra-create-openshift-project

```diff
@@ -0,0 +1,19 @@
+PROJECT=continuous-infra
```

Comment:
This file is not necessary we already have contra-env-setup to take the gold images and load them